dist/tools/backport_pr/backport_pr.py

```diff
@@ -0,0 +1,218 @@
+REPO = "RIOT"
+GITHUBTOKEN_FILE = ".riotgithubtoken"
+WORKTREE_SUBDIR = "backport_temp"
+
+RELEASE_SUFFIX = "-branch"
+RELEASE_PREFIX = ""
+
+LABELS_REMOVE = ['Process: needs backport']
+LABELS_ADD = ['Process: release backport']
+
+BACKPORT_BRANCH = 'backport/{release}/{origbranch}'
+
+
+def _get_labels(pr):
+    labels = {label['name'] for label in pr['labels']}
+    for remove in LABELS_REMOVE:
+        labels.discard(remove)
+    labels.update(LABELS_ADD)
+    return sorted(list(labels))
+
+
+def _branch_name_strip(branch_name):
+    if branch_name.startswith(RELEASE_PREFIX) and \
+       branch_name.endswith(RELEASE_SUFFIX):
+        if len(RELEASE_SUFFIX):
+            branch_name = branch_name.rsplit(RELEASE_SUFFIX, maxsplit=1)[0]
+        if len(RELEASE_PREFIX):
+            branch_name = branch_name.split(RELEASE_PREFIX, maxsplit=1)[0]
+    return branch_name
+
+
+def _get_latest_release(branches):
+    version_latest = 0
+    release_fullname = ''
+    release_short = ''
+    for branch in branches:
+        branch_name = _branch_name_strip(branch['name'])
+        branch_num = 0
+        try:
+            branch_num = int(''.join(branch_name.split('.')))
+        except ValueError:
+            pass
+        if branch_num > version_latest:
+            version_latest = branch_num
+            release_short = branch_name
+            release_fullname = branch['name']
+    return (release_short, release_fullname)
+
+
+def _get_upstream(repo):
+    for remote in repo.remotes:
+        if (remote.url.endswith("{}/{}.git".format(ORG, REPO)) or
+                remote.url.endswith("{}/{}".format(ORG, REPO))):
+            return remote
+
+
+def _delete_worktree(repo, workdir):
+    shutil.rmtree(workdir)
+    repo.git.worktree('prune')
+
+
+def main():
+    keyfile = os.path.join(os.environ['HOME'], GITHUBTOKEN_FILE)
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-k", "--keyfile", type=argparse.FileType('r'),
+                        default=keyfile,
+                        help="File containing github token")
+    parser.add_argument("-c", "--comment", action="store_true",
+                        help="Put a comment with a reference under"
+                             "the original PR")
+    parser.add_argument("-n", "--noop", action="store_true",
+                        help="Limited noop mode, creates branch, but doesn't"
+                             "push and create the PR")
+    parser.add_argument("-r", "--release-branch", type=str,
+                        help="Base the backport on this branch, "
+                             "default is the latest")
+    parser.add_argument("--backport-branch-fmt", type=str,
+                        default=BACKPORT_BRANCH,
+                        help="Backport branch format. "
+                             "Fields '{release}' and '{origbranch} will be "
+                             "replaced by the release name and remote branch "
+                             "name.")
```

Comment:
default doc missing

Reply:
I could do it with 
``` diff
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```
But then other default do not look that great :/

Reply:
How about doing it just like the other defaults for now. Other changes can always come later ;-).